<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Refine</class>
 <widget class="QDialog" name="Refine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DOB From</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gender">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QRadioButton" name="female">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="male">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="mob">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MOB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchByID">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search By ID</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dobFrom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/calendar-1559935_960_720.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>127</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>ID of Player</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="id"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DOB To</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cal_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/calendar-1559935_960_720.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="dobTo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>370</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="whereStatement">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT * FROM PLAYERS WHERE</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>31</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>31</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>)</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="custom">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Custom Search</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>searchByID</tabstop>
  <tabstop>fName</tabstop>
  <tabstop>lName</tabstop>
  <tabstop>dobFrom</tabstop>
  <tabstop>cal</tabstop>
  <tabstop>mob</tabstop>
  <tabstop>email</tabstop>
  <tabstop>female</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>clear</tabstop>
  <tabstop>search</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
